packages/data-models/flowTypes.ts

```diff
@@ -380,6 +380,7 @@ export namespace FlowTypes {
     "audio_end",
     "audio_play",
     "changed",
+    "download",
```

Comment:
nit(blocking) given that the _download_ functionality doesn't necessarily download (e.g. convert to temp object url and open), then would it be worth using a combination of `download` and `open` or `file_open` or `open_external` to make clearer what subset of behaviour expected?

Reply:
I'd probably lean more towards keeping `share` `open` and `download` as standalone actions as I'm not sure grouping them together adds much value (particularly given you can currently share non-files, e.g. text, and in the future we may want to support opening non-files, such as programmatically opening a web url (using browser, instead of in-app navigation). 

Maybe we have top level `share`, `download` and `open` actions which then support some extra param syntax, e.g. `open : asset_path | external:true`